src/honey/sql.cljc

```diff
@@ -214,11 +214,13 @@
      (fn [fk _] (param-value (fk)))}))
 
 (defn- format-var [x & [opts]]
-  (let [c (name-_ x)]
+  (let [c (name x)]
```

Comment:
This change will break things unless you also change the `upper-case` call below to `sql-kw`. Consider:
```clojure
user=> (sql/format {:select [:%char-length.customer-name]})
["SELECT CHAR_LENGTH(customer_name)"]
```
At least one test should use kebab-case for both the function name and at least one column argument.

Reply:
Just realized I was wrong about this -- `sql-kw` will replace `-` with space. Sorry.